assets/stylesheets/_components.scss

```diff
@@ -10,9 +10,10 @@
 
 @import 'auth/style';
 @import 'blocks/author-compact-profile/style';
+@import 'blocks/author-selector/style';
 @import 'blocks/comment-button/style';
 @import 'blocks/comments/style';
-@import 'blocks/author-selector/style';
+@import 'blocks/dismissable-card/style';
```

Comment:
This is an interesting one. Given it's fairly generic I could expect it to be in `components` but the connection with user preferences throws a bit of wrench. :)